presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -3126,8 +3126,10 @@ public void testIn()
             throws Exception
     {
         assertQuery("SELECT orderkey FROM orders WHERE orderkey IN (1, 2, 3)");
+        assertQuery(String.format("SELECT orderkey FROM orders WHERE orderkey IN (%d, %d, %d)", Integer.MAX_VALUE + 1L, Integer.MAX_VALUE + 2L, Integer.MAX_VALUE + 3L));
```

Comment:
TPCH is very limited. You will notice that if you try `SELECT max(orderkey) FROM orders`. We've been considering using a better generated test set for a while. For the time being, I believe these three tests are better.
```
SELECT x FROM (values 3, 100) t(x) WHERE x IN (2147483649)
SELECT x FROM (values 3, 100) t(x) WHERE x IN (3, cast(rand() * 12345678910 as bigint))
SELECT x FROM (values 3, 100, 2147483648, 2147483649, 2147483650) t(x) WHERE x IN (2147483648, 2147483650)
SELECT x FROM (values 3, 100, 2147483648, 2147483649, 2147483650) t(x) WHERE x IN (3, 4, 2147483648, 2147483650)
```